soh/soh/GameMenuBar.cpp

```diff
@@ -1417,9 +1417,9 @@ namespace GameMenuBar {
             ImGui::PushStyleVar(ImGuiStyleVar_FrameBorderSize, 1.0f);
             ImGui::PushStyleColor(ImGuiCol_Border, ImVec4(0.22f, 0.38f, 0.56f, 1.0f));
         #ifdef __WIIU__
-            static ImVec2 buttonSize(160.0f * 2.0f, 0.0f);
+            static ImVec2 buttonSize(195.0f * 2.0f, 0.0f);
         #else
-            static ImVec2 buttonSize(160.0f, 0.0f);
+            static ImVec2 buttonSize(195.0f, 0.0f);
```

Comment:
This doesn't account for when fast file select is enabled, which makes it slightly larger than the debug warp change did. I was able to get to full width with:
```suggestion
            static ImVec2 buttonSize(7.0f - ImGui::GetContentRegionMax().x, 0.0f);
```

not sure if you'd multiple that whole x value by 2 for WiiU or just the 7..

Reply:
Turns out your solution worked because it resolved as -1.0f, which just happened to be something that ImGui looks at like "make this fit the space the button can possibly fill".